621.py

```diff
@@ -1,20 +1,13 @@
 class Solution:
     def leastInterval(self, tasks: List[str], n: int) -> int:
         task_freqs = Counter(tasks)
-        sorted_freqs = sorted(task_freqs.values(), reverse=True)
-
-        max_freq, max_count = sorted_freqs[0], 1
-
-        for freq in sorted_freqs[1:]:
-            if freq == max_freq:
-                max_count += 1
-            else:
-                break
-
-        part_count = max_freq - 1
-        part_length = n - (max_count - 1)
-        empty_slots = part_count * part_length
-        left_task_slots = len(tasks) - max_freq * max_count
-        idles = max(0, empty_slots - left_task_slots)
+        max_freq = max(task_freqs.values())
+        max_count = list(task_freqs.values()).count(max_freq)
```

Comment:
`max_count` should really be `max_freq_count`. `max_count` alone doesn't mean anything, i.e., maximum count of what?